tests/unit/routers/test_platform.py

```diff
@@ -99,9 +99,9 @@ def test_read_health(anon_client):
     assert resp.status_code == 200
 
 
-def test_get_metrics(anon_client, setup_metrics):
+def test_get_metrics(anon_client, setup_metrics, caplog):
     """An anonoymous user can request metrics."""
-    with capture_logs() as cap_logs:
+    with caplog.at_level(logging.INFO, logger="ctms.web"):
         resp = anon_client.get("/metrics")
     assert resp.status_code == 200
-    assert len(cap_logs) == 1
+    assert len(caplog.messages) == 1
```

Comment:
Optional: Do you think these assertions about the number of logs that are emitted are helpful? Maybe we can remove them.

Reply:
I added a small assertion on status code, just to keep it tight